cuqi/testproblem/_testproblem.py

```diff
@@ -254,6 +297,94 @@ def __init__(self,
         self.exactData = y_exact
         self.infoString = "Noise type: Additive {} with std: {}".format(noise_type.capitalize(),noise_std)
 
+def _getConvolutionOperator(dim, PSF, PSF_param, PSF_size, BC):
+
+    # Boundary condition translation
+    if BC.lower() == "zero":
+        mode = "constant"
+    elif BC.lower() == "periodic":
+        mode = "wrap"
+    elif BC.lower() == "Neumann":
+        mode = "symmetric"
+    elif BC.lower() == "Mirror":
```

Comment:
Same here, lower case m?
```suggestion
    elif BC.lower() == "mirror":
```